cypress/integration/inventory/moving-items.spec.js

```diff
@@ -0,0 +1,23 @@
+import TopMenu from '../../support/fragments/topMenu';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import testTypes from '../../support/dictionary/testTypes';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+
+describe('inventory: moving items', () => {
+  beforeEach('navigates to Inventory', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(TopMenu.inventoryPath);
+  });
+
+  it('C15185 verifies moving multiple items between holding', { tags: [testTypes.smoke] }, () => {
```

Comment:
1. Test in testrail related with Holdings part of inventory. Your test related with Instances. Please double check this
2. I can't see verification of movement(just appearing of message from Folio). It is enough?

Reply:
1. You won't able moving item between holdings inside Holding page, so in test case 'View the holdings' means expand this tabs
![image](https://user-images.githubusercontent.com/26041116/151559525-d1b01bc2-9158-4e0c-bf87-e9a022593859.png)

2. I moved item to another holding, then I return it to previous holding. And if item isn't moved I will not return it to previous holding. I think it is enough for this case with checking success message